flask_zipkin.py

```diff
@@ -123,6 +124,13 @@ def create_http_headers_for_new_span(self):
         return zipkin.create_http_headers_for_new_span()
 
     def logging(self, **kwargs):
-        if hasattr(g, '_zipkin_span') and g._zipkin_span and g._zipkin_span.logging_context:
+        logging.warning('This method has been depreated, '
+                        'please call `update_annotations` instead.')
+        self.update_annotations(**kwargs)
+
+    def update_annotations(self, **kwargs):
```

Comment:
The term annotation is most often used for annotations in the zipkin model (timestamp -> string)
if you want a shorter word for binary_annotations, I'd use "update_tags"

for example, we call kv tag in brave.

https://github.com/openzipkin/brave/blob/master/brave/src/main/java/brave/Span.java#L88